Pipfile

```diff
@@ -12,14 +12,15 @@ isort = "*"
 
 [packages]
 
-Flask = "*"
-Flask-Table = "*"
-Flask-RESTful = "*"
+flask = "*"
+flask-table = "*"
```

Comment:
I think this also might be a hold-over from before

Reply:
@henryrizzi You are right about all the obsolete (or aspirational) deps. I can clean that up before I merge.